docs/introduction/how-tos/quickstart.md

```diff
@@ -0,0 +1,66 @@
+Quickstart Guide
+================
+
+Local Install
+----------------
+
+### 1. Run minikube.sh
+
+```bash
+deployment/minikube/minikube.sh
+```
+
+### 2. Prepare Siembol Config Repository
+
+#### 1. Fork Siembol Config Repository
+
+1. Go to https://github.com/G-Research/siembol-config
+1. Fork into your own organization or personal account
+
+#### 2. Update application.properties
+
+Edit `deployment/helm/config-editor/resources/application.properties`
```

Comment:
True, but I'm going to punt this for the moment since I think Marian is champing at the bit to fix the whole application.properties issue.

Reply:
Similarly, we need to deal with `src/siembol/resources/git.config`.

Reply:
feel free to ignore this git config for now since an upgrade of jgit dependency changed where the file should be copied and moreover we need to add one env variable for an internal jgit config  - this way we will later update the charts and intro docs

Reply:
Thanks for the tip.  git.config is now removed and the chart looks fine.